README.md

```diff
@@ -136,6 +135,14 @@ require'nvim-tree'.setup { -- BEGIN_DEFAULT_OPTS
     number = false,
     relativenumber = false,
     signcolumn = "yes",
+    indent_markers = {
```

Comment:
do you think it would make more sens to move the option in a new `renderer.indent_markers` ?

Reply:
That works. All the renderer config :g can go in there.

I'll update on the weekend.